numba/tests/test_npdatetime.py

```diff
@@ -544,6 +550,21 @@ def check(a):
                              (TD,) * len(TD_CASES) + (DT,) * len(TD_CASES)):
             check(typ(*case))
 
+    def test_min_max(self):
+        min_f = self.jit(min_usecase)
+        max_f = self.jit(max_usecase)
+        def check(a, b):
+            self.assertPreciseEqual(min_f(a, b), min(a, b))
+            self.assertPreciseEqual(max_f(a, b), max(a, b))
+
+        check(TD(1), TD(2))
```

Comment:
I think you could simplify this could by have an iterable of arguments and call check on the pairs.

For example:

```python
for x,y in pairs
    check(x, y)
```